test/mock-content-catalog.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const { posix: path } = require('path')
+
+const SPACE_RX = / /g
+
+function mockContentCatalog (seed = []) {
```

Comment:
I really like the API, it's simple to use but I'm a bit concerned about the maintenance.

Is the above code copied/pasted from Antora internal functions? If so could you please add a link to the original code? 
Also could we delegate even more work to Antora?

Reply:
I've put a link to the Antora code and reference to the license and copyright holders in the file. 
I'm not sure how we could delegate more work to Antora but happy to take suggestions.

Reply:
Thanks I will take a look at the original code.